ddp-workspace/projects/ddp-prion/src/index.html

```diff
@@ -24,20 +24,9 @@
 
   <link href="https://fonts.googleapis.com/css?family=Raleway:300i,300,400,500,700" rel="stylesheet">
   <script src="https://kit.fontawesome.com/6ff953c590.js" crossorigin="anonymous"></script>
-  <script type="text/javascript">
```

Comment:
Still waiting to hear back on whether TCell counts as a functional cookie, but there's a significant chance it will count as a functional cookie.

Reply:
I've returned back TCell script just in case it will be counted as functional cookie  in d32188c